packages/components/src/theme/ComposerTheme/Spacing/spacing.ts

```diff
@@ -29,5 +30,16 @@ export const defaultSpacing = {
   'spacer-72': '18rem',
   'spacer-80': '20rem',
   'spacer-86': '24rem',
-  'spacer-0': '0rem',
+  'spacer-button-gap-L': '0.500rem',
+  'spacer-button-y-L': '0.750rem',
+  'spacer-button-x-L': '1.500rem',
+  'spacer-button-gap-M': '0.500rem',
+  'spacer-button-y-M': '0.500rem',
+  'spacer-button-x-M': '1.000rem',
+  'spacer-button-gap-S': '0.500rem',
+  'spacer-button-y-S': '0.500rem',
+  'spacer-button-x-S': '0.750rem',
+  'spacer-button-gap-XS': '0.500rem',
+  'spacer-button-y-XS': '0.250rem',
+  'spacer-button-x-XS': '0.500rem',
```

Comment:
nit: can we remove "extra 0s" at the end of each value?

Reply:
These spacings are auto-generated with the figma file. I believe we have a fixed of 3 decimals because design has some spacings for example as `0.0625`

Reply:
I believe this is a flaky test because it has happened in other [PRs](https://www.chromatic.com/build?appId=5eed577398d08d0022b52d76&number=2733) and it doesn't seem related to this one. What do you think?